src/types/gameConsole.ts

```diff
@@ -50,6 +50,10 @@ interface OutputTokens {
   // @see https://github.com/XargonWan/RetroDECK/blob/main/es-configs/es_systems.xml
   retrodeck?: string,
 
+  // RomM ROMs go in the /roms/{romm} directory:
```

Comment:
I think it's actually:

```suggestion
  // RomM ROMs go in the /romm/library/{romm} directory:
```